packages/cli/src/prompts/method-params.ts

```diff
@@ -125,3 +135,30 @@ function getCommandProps(
     ...args,
   };
 }
+
+function getPlaceholder(type: string): string {
+  const ARRAY_TYPE_REGEX = /(.+)\[\d*\]$/; // matches array type identifiers like uint[] or byte[4]
+
+  if (type.match(ARRAY_TYPE_REGEX)) {
+    const arrayType = type.match(ARRAY_TYPE_REGEX)[1];
+    const itemPlaceholder = getPlaceholder(arrayType);
+    return `[${itemPlaceholder}, ${itemPlaceholder}]`;
```

Comment:
Can we show the simplified version of array arguments? (I recall that you implemented the `parseArg` function to work fine without using `[]`, right?)

Reply:
Yeah, but that fails when you have nested arrays. This one works always, so I decided to go with it.